src/rgw/rgw_op.cc

```diff
@@ -119,8 +121,8 @@ static int parse_range(const char *range, off_t& ofs, off_t& end, bool *partial_
 }
 
 static int decode_policy(CephContext *cct,
-                         bufferlist& bl,
```

Comment:
Do we really need to touch the indentation here and across couple of lines below?